_docs/updating-files-managed-with-modulesync.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: Updating Files Managed With ModuleSync
+date: 2021-06-30
+summary: How to proceed to update files managed by ModuleSync in Vox Pupuli templates.
+---
+
+## What is ModuleSync?
+
+[ModuleSync] is used at Vox Pupuli to syncronise common files across the various modules we manage.  It helps us to update all our modules a consistent way.
```

Comment:
"all of the modules in a consistent way."

Reply:
Fixed, thanks!